avalon/tools/cbloader/widgets.py

```diff
@@ -10,6 +10,7 @@
 from .model import SubsetsModel
 from .delegates import PrettyTimeDelegate, VersionDelegate
 from . import lib
+from .proxy import FilterProxyModel
```

Comment:
This import isn't used. This also means the added `proxy.py` file is redundant in `cbloader`.